elasticsearch/azuredeploy.parameters.json

```diff
@@ -3,13 +3,13 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "dnsNameforLBIP": {
-      "value": "changeme"
+      "value": "elasticlbip"
     },
     "storageAccountPrefix": {
-      "value": "changeme"
+      "value": "elasticstore"
     },
     "adminUsername": {
-      "value": "trent"
+      "value": "admin"
     },
     "adminPassword": {
       "value": "changeme"
```

Comment:
Can you change this value to something that passes password requirements, something like Chang3m3 maybe?